### DISCUSSION TITLE
v0.4.0

### DISCUSSION THREAD

**Discussion by Huskydog9988** [waldo-vision/waldo#471] [Announcements]
## What's Changed
As part of this release we are making some drastic changes to Waldo. We recently [cleaned up our Discord](https://discord.com/channels/903349717323382814/903350180009619467/1187528464656498778), and are in the process of migrating away from Kubernetes, and onto Docker Swarm to minimize tech debt. As part of this change we're migrating our current database CockroachDB, to the much simpler PostgreSQL to help further reduce complexity. 

We've also had some wonderful progress on the model by @mattolson93, hopefully some more details on that soon.

* update staging branch by @Huskydog9988 in https://github.com/waldo-vision/waldo/pull/298
* Removed API link for USER role in Account page & Trusted can now create API keys by @Bensikrac in https://github.com/waldo-vision/waldo/pull/315
* Fix link api percentage limit inverted bug & Database vote query count error by @Bensikrac in https://github.com/waldo-vision/waldo/pull/316
* fix(plausible): set custom domain by @Huskydog9988 in https://github.com/waldo-vision/waldo/pull/332
* Type turnstile ref by @marsidev in https://github.com/waldo-vision/waldo/pull/335
* Replaced React image with Chakra image to fix server error by @Bensikrac in https://github.com/waldo-vision/waldo/pull/440
* Create CODEOWNERS by @WaldoVisionOfficial in https://github.com/waldo-vision/waldo/pull/452
* migrate to postgres provider by @ceriddenn in https://github.com/waldo-vision/waldo/pull/467

## New Contributors
* @marsidev made their first contribution in https://github.com/waldo-vision/waldo/pull/335
* @WaldoVisionOfficial made their first contribution in https://github.com/waldo-vision/waldo/pull/452

**Full Changelog**: https://github.com/waldo-vision/waldo/compare/v0.3.6...v0.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/waldo-vision/waldo/releases/tag/v0.4.0'>v0.4.0</a>.</em>

**Comment by jaredb1011** [comment]
Thanks for posting this update
